fight/fight.go

```diff
@@ -0,0 +1,18 @@
+package rps
```

Comment:
This is in the fight directory so your package should be

```suggestion
package fight
```